etc/inc/disable-programs.inc

```diff
@@ -758,6 +758,8 @@ blacklist ${HOME}/.neverball
 blacklist ${HOME}/.newsbeuter
 blacklist ${HOME}/.newsboat
 blacklist ${HOME}/.nicotine
+blacklist ${HOME}/.npm
+blacklist ${HOME}/.npmrc
```

Comment:
A `noblacklist` for these in `allow-common-devel.inc` would be good, so IDEs don't break.

Reply:
I considered that, but I thought IDEs would be out of scope for adding an npm profile.